frontend-react/src/components/ManagePublicKey/ManagePublicKeyChooseSender.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import { Form, Dropdown, Button, FormGroup } from "@trussworks/react-uswds";
+
+import { UseOrganizationSenders } from "../../hooks/UseOrganizationSenders";
+import Spinner from "../Spinner";
+
+type Props = {
+    onSenderSelect: any; //What type should this be???
+};
+
+export function ManagePublicKeyChooseSender({ onSenderSelect }: Props) {
```

Comment:
One pattern I'd like us to use is exporting default components when it makes sense:

```tsx
export default function ManagePublicKeyChooseSender(...) { ... }
```

This pattern encourages that we orient each file around a _single_ component to locate them more easily and cut down on number of lines in a file (which gets unwieldy after a certain threshold).

Reply:
Agree, my bad.